config/config_test.go

```diff
@@ -19,6 +19,8 @@ tls_cert_file: /opt/tls.cert
 tls_key_file:  /opt/tls.key
 disable_http_ac_validation: true
 enable_endpoint_metrics: true
+read_timeout: 3
+write_timeout: 4
```

Comment:
These example values seem fairly low (at least compared to the current "no limit"). It might be worth adding a brief description here about how admins might choose suitable values.

Reply:
Setting to some real world values :+1: